package.json

```diff
@@ -38,25 +39,50 @@
     "karma-jasmine": "1.1.0",
     "lodash": "4.17.4",
     "mousetrap": "1.4.6",
-    "neutrino": "4.3.1",
-    "neutrino-lint-base": "4.3.1",
-    "neutrino-preset-karma": "4.2.0",
-    "neutrino-preset-react": "4.2.3",
+    "neutrino": "^5.2.0",
+    "neutrino-middleware-copy": "^5.2.0",
+    "neutrino-middleware-eslint": "^5.2.0",
+    "neutrino-preset-karma": "^5.2.0",
+    "neutrino-preset-react": "^5.2.0",
     "ngreact": "0.3.0",
     "raw-loader": "0.5.1",
     "react": "15.3.1",
     "react-addons-test-utils": "15.3.1",
     "react-dom": "15.3.1",
     "taskcluster-client": "1.6.3",
+    "webpack": "^2.3.2",
     "webpack-md5-hash": "0.0.5"
   },
+  "neutrino": {
+    "use": [
+      "./neutrino-custom/lint.js",
+      "neutrino-preset-react",
+      "./neutrino-custom"
+    ],
+    "options": {
+      "source": "ui",
+      "output": "dist",
+      "entry": "entry-index.js",
+      "tests": "tests",
+      "compile": {
+        "targets": {
+          "browsers": [
+            "last 1 Chrome versions",
+            "last 1 Firefox versions",
+            "last 1 Edge versions",
+            "last 1 Safari versions"
+          ]
+        }
+      }
+    }
+  },
   "scripts": {
-    "build": "./node_modules/neutrino/bin/neutrino build --presets ./neutrino-custom/production.js",
-    "lint": "./node_modules/eslint/bin/eslint.js ui/",
-    "start": "./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
-    "start:local": "./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/local-watch.js",
-    "start:stage": "SERVICE_DOMAIN=https://treeherder.allizom.org ./node_modules/neutrino/bin/neutrino start --presets ./neutrino-custom/development.js",
-    "test": "./node_modules/neutrino/bin/neutrino test --presets ./neutrino-custom/test.js",
-    "test:watch": "./node_modules/neutrino/bin/neutrino test --watch --presets ./neutrino-custom/test.js"
+    "build": "./node_modules/neutrino/bin/neutrino build",
+    "lint": "NODE_ENV=development ./node_modules/eslint/bin/eslint.js --ext .js,.jsx ui/",
```

Comment:
> Regarding ext, right now they must be passed on the command line, and do not have the ability to be set in the config file.

I was going to file an eslint issue for adding an .eslintrc option, but looks like there is a reason why it's not possible:
https://github.com/eslint/eslint/issues/3469

That said, perhaps they could default to including .jsx, to avoid the need to override. I've filed:
https://github.com/eslint/eslint/issues/8399